stats/views.py

```diff
@@ -24,34 +24,35 @@ def stats(request):
     cards = []
     values = [f"{lang.code}_accounts" for lang in (LANGUAGES + FRAMEWORKS)] + ["date"]
     daily_objects = Daily.objects.order_by("-date").values(*values)[:graph_days_len]
-    for lang in LANGUAGES + FRAMEWORKS:
-        card = {
-            "code": lang.code,
-            "name": lang.name,
-            "emoji": lang.emoji,
-            "image": lang.image,
-            "accounts_count": [],
-            "dates": [],
-            "percent_change": 0,
-            "total_accounts": 0,
-        }
+    if len(daily_objects) > 0:
```

Comment:
```suggestion
    if daily_objects:
```

Empty queryset is falsy so you don't have to check the length.